From 3579072a1c9e703fb1096d5c5eafa81509c6f437 Mon Sep 17 00:00:00 2001
From: Hiroshi DOYU <Hiroshi.DOYU@nokia.com>
Date: Sat, 13 Sep 2008 11:33:40 +0300
Subject: [PATCH] BRIDGE: Fix incorrect pathname

sed -i -e \
's@bridge/inc@linux/arch/arm/plat-omap/include/dspbridge/@'\
*.h

Signed-off-by: Hiroshi DOYU <Hiroshi.DOYU@nokia.com>
---
 arch/arm/plat-omap/include/dspbridge/chnl_sm.h     |    2 +-
 arch/arm/plat-omap/include/dspbridge/dmm.h         |    2 +-
 arch/arm/plat-omap/include/dspbridge/dpc.h         |    2 +-
 arch/arm/plat-omap/include/dspbridge/drv.h         |    2 +-
 arch/arm/plat-omap/include/dspbridge/drvdefs.h     |    2 +-
 arch/arm/plat-omap/include/dspbridge/dspdrv.h      |    2 +-
 .../plat-omap/include/dspbridge/dynamic_loader.h   |    2 +-
 arch/arm/plat-omap/include/dspbridge/errbase.h     |    2 +-
 arch/arm/plat-omap/include/dspbridge/gb.h          |    2 +-
 arch/arm/plat-omap/include/dspbridge/getsection.h  |    2 +-
 arch/arm/plat-omap/include/dspbridge/gh.h          |    2 +-
 arch/arm/plat-omap/include/dspbridge/gs.h          |    2 +-
 arch/arm/plat-omap/include/dspbridge/gt.h          |    2 +-
 arch/arm/plat-omap/include/dspbridge/host_os.h     |    2 +-
 arch/arm/plat-omap/include/dspbridge/io.h          |    2 +-
 arch/arm/plat-omap/include/dspbridge/io_sm.h       |    2 +-
 arch/arm/plat-omap/include/dspbridge/iodefs.h      |    2 +-
 arch/arm/plat-omap/include/dspbridge/kfile.h       |    2 +-
 arch/arm/plat-omap/include/dspbridge/ldr.h         |    2 +-
 arch/arm/plat-omap/include/dspbridge/list.h        |    2 +-
 arch/arm/plat-omap/include/dspbridge/mbx_sh.h      |    2 +-
 arch/arm/plat-omap/include/dspbridge/mem.h         |    2 +-
 arch/arm/plat-omap/include/dspbridge/memdefs.h     |    2 +-
 arch/arm/plat-omap/include/dspbridge/mgr.h         |    2 +-
 arch/arm/plat-omap/include/dspbridge/mgrpriv.h     |    2 +-
 arch/arm/plat-omap/include/dspbridge/msg.h         |    2 +-
 arch/arm/plat-omap/include/dspbridge/msgdefs.h     |    2 +-
 arch/arm/plat-omap/include/dspbridge/nldr.h        |    2 +-
 arch/arm/plat-omap/include/dspbridge/nldrdefs.h    |    2 +-
 arch/arm/plat-omap/include/dspbridge/node.h        |    2 +-
 arch/arm/plat-omap/include/dspbridge/nodedefs.h    |    2 +-
 arch/arm/plat-omap/include/dspbridge/nodepriv.h    |    2 +-
 arch/arm/plat-omap/include/dspbridge/ntfy.h        |    2 +-
 arch/arm/plat-omap/include/dspbridge/proc.h        |    2 +-
 34 files changed, 34 insertions(+), 34 deletions(-)

--- a/arch/arm/plat-omap/include/dspbridge/chnl_sm.h
+++ b/arch/arm/plat-omap/include/dspbridge/chnl_sm.h
@@ -1,5 +1,5 @@
 /*
- * dspbridge/inc/chnl_sm.h
+ * dsplinux/arch/arm/plat-omap/include/dspbridge//chnl_sm.h
  *
  * DSP-BIOS Bridge driver support functions for TI OMAP processors.
  *
--- a/arch/arm/plat-omap/include/dspbridge/dmm.h
+++ b/arch/arm/plat-omap/include/dspbridge/dmm.h
@@ -1,5 +1,5 @@
 /*
- * bridge/inc/dmm.h
+ * linux/arch/arm/plat-omap/include/dspbridge//dmm.h
  *
  * DSP-BIOS Bridge driver support functions for TI OMAP processors.
  *
--- a/arch/arm/plat-omap/include/dspbridge/dpc.h
+++ b/arch/arm/plat-omap/include/dspbridge/dpc.h
@@ -1,5 +1,5 @@
 /*
- * bridge/inc/dpc.h
+ * linux/arch/arm/plat-omap/include/dspbridge//dpc.h
  *
  * DSP-BIOS Bridge driver support functions for TI OMAP processors.
  *
--- a/arch/arm/plat-omap/include/dspbridge/drv.h
+++ b/arch/arm/plat-omap/include/dspbridge/drv.h
@@ -1,5 +1,5 @@
 /*
- * bridge/inc/drv.h
+ * linux/arch/arm/plat-omap/include/dspbridge//drv.h
  *
  * DSP-BIOS Bridge driver support functions for TI OMAP processors.
  *
--- a/arch/arm/plat-omap/include/dspbridge/drvdefs.h
+++ b/arch/arm/plat-omap/include/dspbridge/drvdefs.h
@@ -1,5 +1,5 @@
 /*
- * bridge/inc/drvdefs.h
+ * linux/arch/arm/plat-omap/include/dspbridge//drvdefs.h
  *
  * DSP-BIOS Bridge driver support functions for TI OMAP processors.
  *
--- a/arch/arm/plat-omap/include/dspbridge/dspdrv.h
+++ b/arch/arm/plat-omap/include/dspbridge/dspdrv.h
@@ -1,5 +1,5 @@
 /*
- * bridge/inc/dspdrv.h
+ * linux/arch/arm/plat-omap/include/dspbridge//dspdrv.h
  *
  * DSP-BIOS Bridge driver support functions for TI OMAP processors.
  *
--- a/arch/arm/plat-omap/include/dspbridge/dynamic_loader.h
+++ b/arch/arm/plat-omap/include/dspbridge/dynamic_loader.h
@@ -1,5 +1,5 @@
 /*
- * bridge/inc/dynamic_loader.h
+ * linux/arch/arm/plat-omap/include/dspbridge//dynamic_loader.h
  *
  * DSP-BIOS Bridge driver support functions for TI OMAP processors.
  *
--- a/arch/arm/plat-omap/include/dspbridge/errbase.h
+++ b/arch/arm/plat-omap/include/dspbridge/errbase.h
@@ -1,5 +1,5 @@
 /*
- * bridge/inc/errbase.h
+ * linux/arch/arm/plat-omap/include/dspbridge//errbase.h
  *
  * DSP-BIOS Bridge driver support functions for TI OMAP processors.
  *
--- a/arch/arm/plat-omap/include/dspbridge/gb.h
+++ b/arch/arm/plat-omap/include/dspbridge/gb.h
@@ -1,5 +1,5 @@
 /*
- * bridge/inc/gb.h
+ * linux/arch/arm/plat-omap/include/dspbridge//gb.h
  *
  * DSP-BIOS Bridge driver support functions for TI OMAP processors.
  *
--- a/arch/arm/plat-omap/include/dspbridge/getsection.h
+++ b/arch/arm/plat-omap/include/dspbridge/getsection.h
@@ -1,5 +1,5 @@
 /*
- * bridge/inc/getsection.h
+ * linux/arch/arm/plat-omap/include/dspbridge//getsection.h
  *
  * DSP-BIOS Bridge driver support functions for TI OMAP processors.
  *
--- a/arch/arm/plat-omap/include/dspbridge/gh.h
+++ b/arch/arm/plat-omap/include/dspbridge/gh.h
@@ -1,5 +1,5 @@
 /*
- * bridge/inc/gh.h
+ * linux/arch/arm/plat-omap/include/dspbridge//gh.h
  *
  * DSP-BIOS Bridge driver support functions for TI OMAP processors.
  *
--- a/arch/arm/plat-omap/include/dspbridge/gs.h
+++ b/arch/arm/plat-omap/include/dspbridge/gs.h
@@ -1,5 +1,5 @@
 /*
- * bridge/inc/gs.h
+ * linux/arch/arm/plat-omap/include/dspbridge//gs.h
  *
  * DSP-BIOS Bridge driver support functions for TI OMAP processors.
  *
--- a/arch/arm/plat-omap/include/dspbridge/gt.h
+++ b/arch/arm/plat-omap/include/dspbridge/gt.h
@@ -1,5 +1,5 @@
 /*
- * bridge/inc/gt.h
+ * linux/arch/arm/plat-omap/include/dspbridge//gt.h
  *
  * DSP-BIOS Bridge driver support functions for TI OMAP processors.
  *
--- a/arch/arm/plat-omap/include/dspbridge/host_os.h
+++ b/arch/arm/plat-omap/include/dspbridge/host_os.h
@@ -1,5 +1,5 @@
 /*
- * bridge/inc/host_os.h
+ * linux/arch/arm/plat-omap/include/dspbridge//host_os.h
  *
  * DSP-BIOS Bridge driver support functions for TI OMAP processors.
  *
--- a/arch/arm/plat-omap/include/dspbridge/io.h
+++ b/arch/arm/plat-omap/include/dspbridge/io.h
@@ -1,5 +1,5 @@
 /*
- * bridge/inc/io.h
+ * linux/arch/arm/plat-omap/include/dspbridge//io.h
  *
  * DSP-BIOS Bridge driver support functions for TI OMAP processors.
  *
--- a/arch/arm/plat-omap/include/dspbridge/io_sm.h
+++ b/arch/arm/plat-omap/include/dspbridge/io_sm.h
@@ -1,5 +1,5 @@
 /*
- * bridge/inc/io_sm.h
+ * linux/arch/arm/plat-omap/include/dspbridge//io_sm.h
  *
  * DSP-BIOS Bridge driver support functions for TI OMAP processors.
  *
--- a/arch/arm/plat-omap/include/dspbridge/iodefs.h
+++ b/arch/arm/plat-omap/include/dspbridge/iodefs.h
@@ -1,5 +1,5 @@
 /*
- * bridge/inc/iodefs.h
+ * linux/arch/arm/plat-omap/include/dspbridge//iodefs.h
  *
  * DSP-BIOS Bridge driver support functions for TI OMAP processors.
  *
--- a/arch/arm/plat-omap/include/dspbridge/kfile.h
+++ b/arch/arm/plat-omap/include/dspbridge/kfile.h
@@ -1,5 +1,5 @@
 /*
- * bridge/inc/kfile.h
+ * linux/arch/arm/plat-omap/include/dspbridge//kfile.h
  *
  * DSP-BIOS Bridge driver support functions for TI OMAP processors.
  *
--- a/arch/arm/plat-omap/include/dspbridge/ldr.h
+++ b/arch/arm/plat-omap/include/dspbridge/ldr.h
@@ -1,5 +1,5 @@
 /*
- * bridge/inc/ldr.h
+ * linux/arch/arm/plat-omap/include/dspbridge//ldr.h
  *
  * DSP-BIOS Bridge driver support functions for TI OMAP processors.
  *
--- a/arch/arm/plat-omap/include/dspbridge/list.h
+++ b/arch/arm/plat-omap/include/dspbridge/list.h
@@ -1,5 +1,5 @@
 /*
- * bridge/inc/list.h
+ * linux/arch/arm/plat-omap/include/dspbridge//list.h
  *
  * DSP-BIOS Bridge driver support functions for TI OMAP processors.
  *
--- a/arch/arm/plat-omap/include/dspbridge/mbx_sh.h
+++ b/arch/arm/plat-omap/include/dspbridge/mbx_sh.h
@@ -1,5 +1,5 @@
 /*
- * bridge/inc/mbx_sh.h
+ * linux/arch/arm/plat-omap/include/dspbridge//mbx_sh.h
  *
  * DSP-BIOS Bridge driver support functions for TI OMAP processors.
  *
--- a/arch/arm/plat-omap/include/dspbridge/mem.h
+++ b/arch/arm/plat-omap/include/dspbridge/mem.h
@@ -1,5 +1,5 @@
 /*
- * bridge/inc/mem.h
+ * linux/arch/arm/plat-omap/include/dspbridge//mem.h
  *
  * DSP-BIOS Bridge driver support functions for TI OMAP processors.
  *
--- a/arch/arm/plat-omap/include/dspbridge/memdefs.h
+++ b/arch/arm/plat-omap/include/dspbridge/memdefs.h
@@ -1,5 +1,5 @@
 /*
- * bridge/inc/memdefs.h
+ * linux/arch/arm/plat-omap/include/dspbridge//memdefs.h
  *
  * DSP-BIOS Bridge driver support functions for TI OMAP processors.
  *
--- a/arch/arm/plat-omap/include/dspbridge/mgr.h
+++ b/arch/arm/plat-omap/include/dspbridge/mgr.h
@@ -1,5 +1,5 @@
 /*
- * bridge/inc/mgr.h
+ * linux/arch/arm/plat-omap/include/dspbridge//mgr.h
  *
  * DSP-BIOS Bridge driver support functions for TI OMAP processors.
  *
--- a/arch/arm/plat-omap/include/dspbridge/mgrpriv.h
+++ b/arch/arm/plat-omap/include/dspbridge/mgrpriv.h
@@ -1,5 +1,5 @@
 /*
- * bridge/inc/mgrpriv.h
+ * linux/arch/arm/plat-omap/include/dspbridge//mgrpriv.h
  *
  * DSP-BIOS Bridge driver support functions for TI OMAP processors.
  *
--- a/arch/arm/plat-omap/include/dspbridge/msg.h
+++ b/arch/arm/plat-omap/include/dspbridge/msg.h
@@ -1,5 +1,5 @@
 /*
- * bridge/inc/msg.h
+ * linux/arch/arm/plat-omap/include/dspbridge//msg.h
  *
  * DSP-BIOS Bridge driver support functions for TI OMAP processors.
  *
--- a/arch/arm/plat-omap/include/dspbridge/msgdefs.h
+++ b/arch/arm/plat-omap/include/dspbridge/msgdefs.h
@@ -1,5 +1,5 @@
 /*
- * bridge/inc/msgdefs.h
+ * linux/arch/arm/plat-omap/include/dspbridge//msgdefs.h
  *
  * DSP-BIOS Bridge driver support functions for TI OMAP processors.
  *
--- a/arch/arm/plat-omap/include/dspbridge/nldr.h
+++ b/arch/arm/plat-omap/include/dspbridge/nldr.h
@@ -1,5 +1,5 @@
 /*
- * bridge/inc/nldr.h
+ * linux/arch/arm/plat-omap/include/dspbridge//nldr.h
  *
  * DSP-BIOS Bridge driver support functions for TI OMAP processors.
  *
--- a/arch/arm/plat-omap/include/dspbridge/nldrdefs.h
+++ b/arch/arm/plat-omap/include/dspbridge/nldrdefs.h
@@ -1,5 +1,5 @@
 /*
- * bridge/inc/nldrdefs.h
+ * linux/arch/arm/plat-omap/include/dspbridge//nldrdefs.h
  *
  * DSP-BIOS Bridge driver support functions for TI OMAP processors.
  *
--- a/arch/arm/plat-omap/include/dspbridge/node.h
+++ b/arch/arm/plat-omap/include/dspbridge/node.h
@@ -1,5 +1,5 @@
 /*
- * bridge/inc/node.h
+ * linux/arch/arm/plat-omap/include/dspbridge//node.h
  *
  * DSP-BIOS Bridge driver support functions for TI OMAP processors.
  *
--- a/arch/arm/plat-omap/include/dspbridge/nodedefs.h
+++ b/arch/arm/plat-omap/include/dspbridge/nodedefs.h
@@ -1,5 +1,5 @@
 /*
- * bridge/inc/nodedefs.h
+ * linux/arch/arm/plat-omap/include/dspbridge//nodedefs.h
  *
  * DSP-BIOS Bridge driver support functions for TI OMAP processors.
  *
--- a/arch/arm/plat-omap/include/dspbridge/nodepriv.h
+++ b/arch/arm/plat-omap/include/dspbridge/nodepriv.h
@@ -1,5 +1,5 @@
 /*
- * bridge/inc/nodepriv.h
+ * linux/arch/arm/plat-omap/include/dspbridge//nodepriv.h
  *
  * DSP-BIOS Bridge driver support functions for TI OMAP processors.
  *
--- a/arch/arm/plat-omap/include/dspbridge/ntfy.h
+++ b/arch/arm/plat-omap/include/dspbridge/ntfy.h
@@ -1,5 +1,5 @@
 /*
- * bridge/inc/ntfy.h
+ * linux/arch/arm/plat-omap/include/dspbridge//ntfy.h
  *
  * DSP-BIOS Bridge driver support functions for TI OMAP processors.
  *
--- a/arch/arm/plat-omap/include/dspbridge/proc.h
+++ b/arch/arm/plat-omap/include/dspbridge/proc.h
@@ -1,5 +1,5 @@
 /*
- * bridge/inc/proc.h
+ * linux/arch/arm/plat-omap/include/dspbridge//proc.h
  *
  * DSP-BIOS Bridge driver support functions for TI OMAP processors.
  *
